docker-compose.yml

```diff
@@ -2,6 +2,7 @@ version: "3"
 
 services:
   studio:
+    platform: linux/amd64
```

Comment:
- Is this configuration not needed for other docker-compose*.yaml?
- Also, does this setting have any negative impact when building on non-macos environments (e.g. Linux)?

Reply:
`platform: linux/amd64` seems to have side effects in non-Apple Silicon builds.
In fact, the following error occurs in the build in Ubuntu environment.

> 12.6 : Failed to fetch http://deb.debian.org/debian/pool/main/f/freetype/libfreetype6_2.12.1%2bdfsg-5_amd64.deb  404  Not Found [IP: 146.75.114.132 80]
> 112.6 E: Failed to fetch http://deb.debian.org/debian-security/pool/updates/main/l/linux/linux-libc-dev_6.1.90-1_amd64.deb  404  Not Found [IP: 146.75.114.132 80]
> 112.6 E: Failed to fetch http://deb.debian.org/debian/pool/main/a/aom/libaom3_3.6.0-1_amd64.deb  404  Not Found [IP: 146.75.114.132 

*The same error continues since then.

Countermeasures are needed.

Reply:
(Additional Note)
After cleaning up the Docker layer cache and starting over from the first step, the build seemed to run successfully.